src/_posts/platform/app/2000-01-01-crash.md

```diff
@@ -1,69 +1,217 @@
-    <li>A crash which occures after 5 minutes has no effect on the cool down</li>
-  </ul>
-  <p>
-    This limitation has been setup in order to avoid the situation when an
-    application try to boot and crashes immediately.
-  </p>
+  A Timeout Error can only occur if you have a `web` or a `tcp` process type,
+  which is very likely.
 {% endnote %}
 
-A limit of __12__ restart operations exist. It means that after 6 hours and 30 minutes,
-your application __will not be accessible anymore__ (The last cool-off duration is 1 hour).
-
-## How do I know my app has crashed?
+#### Fixing Timeout Errors
 
-When such an event occurred, you'll receive notifications, as well as your collaborators for the concerned application. You'll receive a notification after the 2nd, 5th and 12th crashes (following the restart policy detailed previously).
+To fix a Timeout Error, make sure:
+- to bind to the provided network port, by using the `PORT` environment
+  variable.
+- to listen on `0.0.0.0` and not on `127.0.0.1`.
+- that your application is starting quickly enough.
 
-It means that if your app crashed once for a temporary reason, it will be automatically restarted. If it does not crash anymore, you will not receive an email. But if it crashes in a loop, you'll get a notification in the minute.
+You may need to edit your
+[Procfile]({% post_url platform/app/2000-01-01-procfile %}) to fulfill these
+requirements.
 
-You'll also find the 'crash' events in your app timeline on the dashboard.
+If your application doesn't need a `web` or `tcp` process type, make sure to
+[scale the unnecessary process type to zero]({% post_url platform/app/2000-01-01-web-less-app %}#deploy-a-web-less-application).
 
-## What to do?
+### Understanding Hook Errors
 
-We strongly advise you to look at the logs of your application using the web
-dashboard or by using the [CLI tool](http://cli.scalingo.com).
+If your application has a [`postdeploy` process type]({% post_url platform/app/2000-01-01-procfile %}#special-process-types),
+the platform can throw a **Hook Error** if the post-deployment process fails.
 
-According to the information gathered, you should then modify your application to
-fix the issues which lead to this instability.
+- In such a case, the deployment fails with a **hook-error** status.
+- If your application is already running, its code isn't updated and the former
+  version keeps running.
 
 {% note %}
-  After a successful deployment, a manual restart, or scaling of your application, we
-  cancel any queued restart job, and the cool down time after the next crash is resetted.
+  Hook Errors can only occur if you have a `postdeploy` process type.
 {% endnote %}
 
-## Need support?
+#### Fixing Hook Errors
+
+Hook Errors are generally caused by an error in your codebase or by some
+misconfiguration. To recover from it, we first advise to investigate the logs
+of your failed deployment to understand the root cause. After fixing it,
```

Comment:
I think it's more in the application logs rather than on the deployment logs that lies the root cause of the hook error.

Reply:
Fixed in https://github.com/Scalingo/documentation/pull/2049/commits/947b11165d3ed66d2cc0df470a7546e5d1e5c14f. Thanks :)